tests/e2e/lib.sh

```diff
@@ -295,6 +295,9 @@ deploy_sensor() {
         # et al.
         kubectl -n stackrox set resources deploy/sensor -c sensor --requests 'cpu=2' --limits 'cpu=4'
     fi
+    if [[ "${ROX_COMPLIANCE_ENHANCEMENTS}" == "true" ]]; then
```

Comment:
It should no longer be necessary to set the flag on the sensor side after #8889

Reply:
Cool - I'll rebase and remove this to see if I can get a clean run.

Reply:
Good call - that seems to have worked:

```
INFO: Tue Jan  2 23:16:40 UTC 2024: Running compliance v2 e2e tests
make: Entering directory '/go/src/github.com/stackrox/stackrox/tests'
CGO_ENABLED is not 0. Compiling with -linkmode=external
go: downloading github.com/machinebox/graphql v0.2.2
go: downloading github.com/graph-gophers/graphql-go v1.5.0
go: downloading go.etcd.io/bbolt v1.3.8
go: downloading go.uber.org/mock v0.4.0
go: downloading github.com/ComplianceAsCode/compliance-operator v1.4.0
go: downloading sigs.k8s.io/controller-runtime v0.16.3
=== RUN   TestComplianceV2Integration
--- PASS: TestComplianceV2Integration (0.13s)
=== RUN   TestComplianceV2ProfileCount
--- PASS: TestComplianceV2ProfileCount (0.13s)
=== RUN   TestComplianceV2ProfileGet
--- PASS: TestComplianceV2ProfileGet (0.13s)
PASS
	github.com/stackrox/rox/tests	coverage: 0.0% of statements
ok  	github.com/stackrox/rox/tests	0.432s
make[1]: Entering directory '/go/src/github.com/stackrox/stackrox/tests'
+ /go/bin/go-junit-report
go: downloading github.com/jstemmer/go-junit-report/v2 v2.1.0
+ report

Test coverage summary:
```